### PR TITLE
v2v: try import ovirt module and set ovirt to None if not found

### PR DESCRIPTION
**Pull request by xiaodwan**
1) Some output modes don't need ovirt module, it should not be
   mandatory. It's up to users who should prepare it if necessary.
2) Remove unused windows functions.

### CODE REVIEW

**Comment by xiaodwan** [comment]
```
# avocado run --vt-type v2v --test-runner=runner --vt-machine-type q35 convert_vm_to_ovirt.esx.vm.8_0.linux.latest9.boot_uefi.arch_x86_64.raw_f.NFS.rhv_upload.rhv_direct.rhv_noverifypeer.preallocated.it_vddk.non_root --vt-connect-uri qemu:///system
JOB ID     : 9f0276c4600005346753a9bbbb4cfbe9faaf5617
JOB LOG    : /var/lib/avocado/job-results/job-2024-06-03T03.14-9f0276c/job.log
 (1/1) type_specific.io-github-autotest-libvirt.convert_vm_to_ovirt.esx.vm.8_0.linux.latest9.boot_uefi.arch_x86_64.raw_f.NFS.rhv_upload.rhv_direct.rhv_noverifypeer.preallocated.it_vddk.non_root: PASS (277.25 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /var/lib/avocado/job-results/job-2024-06-03T03.14-9f0276c/results.html
JOB TIME   : 277.62 s

# avocado run --vt-type v2v --test-runner=runner --vt-machine-type q35 convert_vm_to_libvirt.esx.vm.8_0.linux.latest9_dev.boot_uefi.arch_x86_64.it_vddk --vt-connect-uri qemu:///system
JOB ID     : abc85a51d0b88897dd224e825336998e353ef01a
JOB LOG    : /var/lib/avocado/job-results/job-2024-06-03T02.34-abc85a5/job.log
 (1/1) type_specific.io-github-autotest-libvirt.convert_vm_to_libvirt.esx.vm.8_0.linux.latest9_dev.boot_uefi.arch_x86_64.it_vddk: PASS (256.37 s)
Not opening readers, lock_server_running not locked
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /var/lib/avocado/job-results/job-2024-06-03T02.34-abc85a5/results.html
JOB TIME   : 256.86 s

```

**Comment by vwu-vera** [comment]
@chunfuwen Please help to review and merge this PR. Thanks.:)

File: virttest/utils_v2v.py [hunk 1]
```diff
@@ -25,7 +25,11 @@
 
 from virttest import data_dir
 from virttest import libvirt_vm as lvirt
-from virttest import ovirt, ppm_utils
+
+try:
+    from virttest import ovirt
+except ModuleNotFoundError:
+    ovirt = None
```
Comment (chunfuwen) [inline virttest/utils_v2v.py:32]: hardly see one module is set to None, so maybe just logging out something

File: virttest/utils_v2v.py [hunk 2]
```diff
@@ -25,7 +25,11 @@
 
 from virttest import data_dir
 from virttest import libvirt_vm as lvirt
-from virttest import ovirt, ppm_utils
```
Comment (chunfuwen) [inline virttest/utils_v2v.py:28]: How about ppm_utils module? Is it not necessary as well?

File: virttest/utils_v2v.py (hunk 1 above)
Comment (xiaodwan) [reply virttest/utils_v2v.py:32]: No, the ovirt variant is used in the code. We must define the variant first.
It not, syntax error will be reported by CI.

File: virttest/utils_v2v.py (hunk 2 above)
Comment (xiaodwan) [reply virttest/utils_v2v.py:28]: v2v don't use it any more.

**Comment by chunfuwen** [comment]
small changes, so mark it as minor_change

**Pull request merged**